### PR TITLE
Page object

### PR DESCRIPTION
**Pull request by amiordan**

### CODE REVIEW

File: src/main/java/pages/ProductPage.java [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+
+    @FindBy(xpath = "//span[@class='cart-label']")
+    private WebElement ShoppingCartLink;
+
+    @FindBy(xpath = "//span[@class='wishlist-label']")
+    private WebElement WishListLink;
+
+    public ProductPage(final WebDriver driver) {
         super(driver);
     }
```
Comment (MaximRaicov) [inline src/main/java/pages/ProductPage.java:26]: Why web elements wrapped in the constructor?

File: src/main/java/pages/CategoryPage.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import lombok.Getter;
+
+@Getter
+public class CategoryPage extends PageObject {
+
+    @FindBy(xpath = "//h2[@class='title']//a[contains(text(),'Cell phones')]")
+    private WebElement subcategory;
```
Comment (vdones) [inline src/main/java/pages/CategoryPage.java]: use code convention
rename subcategory in subCategory, please

File: src/main/java/pages/HomePage.java [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
 public class HomePage extends PageObject {
 
-       HomePage(final WebDriver driver) {
+    @FindBy(xpath = "//ul[@class='top-menu notmobile']//a[contains(text(),'Electronics')]")
+    private WebElement category;
+
+    @FindBy(xpath = "//input[@id='small-searchterms']")
+    private WebElement searchField;
+
+    @FindBy(xpath = "//input[@class='button-1 search-box-button']")
```
Comment (vdones) [inline src/main/java/pages/HomePage.java]: change xpath into more explicitly, please

File: src/main/java/pages/LoginPage.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import lombok.Getter;
+
+@Getter
+public class LoginPage extends PageObject {
+
+    @FindBy(xpath = "//input[@id='Email']")
```
Comment (vdones) [inline src/main/java/pages/LoginPage.java]: why you dont use findBy id?

File: src/main/java/pages/MyAccountPage.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import lombok.Getter;
+
+@Getter
+public class MyAccountPage extends PageObject {
+
+    @FindBy(xpath = "//input[@id='FirstName']")
```
Comment (vdones) [inline src/main/java/pages/MyAccountPage.java]: why you dont use findBy id?

File: src/main/java/pages/RegisterPage.java [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import lombok.Getter;
+
+@Getter
+public class RegisterPage extends PageObject {
+
+    @FindBy(xpath = "//input[@id='FirstName']")
```
Comment (vdones) [inline src/main/java/pages/RegisterPage.java]: why you dont use findBy id?

File: src/main/java/pages/CategoryPage.java (hunk 2 above)
Comment (amiordan) [reply src/main/java/pages/CategoryPage.java]: Done

File: src/main/java/pages/HomePage.java (hunk 3 above)
Comment (amiordan) [reply src/main/java/pages/HomePage.java]: Done

File: src/main/java/pages/LoginPage.java (hunk 4 above)
Comment (amiordan) [reply src/main/java/pages/LoginPage.java]: Done

File: src/main/java/pages/MyAccountPage.java (hunk 5 above)
Comment (amiordan) [reply src/main/java/pages/MyAccountPage.java]: Done

File: src/main/java/pages/RegisterPage.java (hunk 6 above)
Comment (amiordan) [reply src/main/java/pages/RegisterPage.java]: Done

File: src/main/java/pages/ProductPage.java (hunk 1 above)
Comment (amiordan) [reply src/main/java/pages/ProductPage.java:26]: they are not wrapped in constructor

**Review by CristianManic — APPROVED** [review]

**Review by CristianManic — APPROVED** [review]

**Review by MaximRaicov — APPROVED** [review]

**Review by anciuvaga — APPROVED** [review]

**Review by vdones — APPROVED** [review]

**Pull request merged**